webpack.config.js

```diff
@@ -44,6 +46,12 @@ module.exports = {
         ]
     },
     plugins: [
+        new webpack.NormalModuleReplacementPlugin(
+            /^node:/u,
+            resource => {
+                resource.request = resource.request.replace(/^node:/u, "");
+            }
+        ),
```

Comment:
This is a solution recommended in https://github.com/webpack/webpack/issues/13290